### PR TITLE
feat: update 117

### PR DESCRIPTION
**Pull request by CjS77**
Ultimately the success of Tari will be driven by the dApplications that are built on top of it, and not based on the 
underlying technology. But without a strong foundation of scalability, security, decentralisation, privacy, and 
amazing developer tools, those dApplications will never be built. Today we discussed how Tari is attempting to 
tackle the scalability portion of that challenge.

### CODE REVIEW

**Review by stringhandler — COMMENTED** [review]
Happy, just think the 5500 eth nodes figure is not from an up to date source

File: _updates/2023-09-11-update-117.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+base layer that focuses on security and manages global state, and a second, digital assets layer that focuses on 
+rapid finalisation and scalability._
+
+_"The distributed system trilemma tells us that these requirements are mutually exclusive._
+
+_"We can't have fast, cheap digital assets and also highly secure and decentralized currency tokens on a single system._
+
+_"Tari overcomes this constraint by building two layers:_
+1. _A base layer that provides a public ledger of Tari coin transactions, secured by PoW to maximize security._
+1. _A DAN consisting of a highly scalable, efficient side-chain that each manages the state of all digital asset."_
```
Comment (SWvheerden) [inline _updates/2023-09-11-update-117.md]:
```suggestion
1. _A DAN consisting of a highly scalable, efficient side-chain that each manages the state of all digital assets."_
```

File: _updates/2023-09-11-update-117.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+only 25 nodes, there will only be a single shard, and so our proto-Cerberus network will run like Aptos or Sui. Every 
+node must reach consensus on every transaction. Since each node running at full-tilt can handle 20 transactions per
+second, that would be the network capacity.
+
+Now let's assume that the network grows to 100 nodes. The network will split into four shards, each with 25 nodes. 
+Assuming the vast majority of transactions involve two shards, the network is effectively running two "cores" and 
+overall capacity is now 40 transactions per second.
+
+In a scenario of "full adoption", there might be 10,000 nodes on the network. This would result in 400 shards, each
+with 25 nodes. This is equivalent of 200 "cores", and overall capacity is now 4,000 transactions per second, all the
```
Comment (SWvheerden) [inline _updates/2023-09-11-update-117.md]:
```suggestion
with 25 nodes. This is equivalent to 200 "cores", and the overall capacity is now 4,000 transactions per second, all the 
```

File: _updates/2023-09-11-update-117.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+Can Tari get to 10,000 or even more nodes? There are currently [5500 Ethereum nodes](https://www.ethernodes.org/) 
+running. Each of these must validate _every transaction_ and maintain a copy of the _entire state_ of the network. 
+This places a huge burden on a node operator. Solana, Aptos, and all the other Layer One smart contract networks 
+have the same problem.
+
+As the Tari network grows and shards become smaller, each validator node will be required to hold _less_ of the overall 
+system's state. 
+Running a Tari node will always require roughly the same level of hardware (excluding storage) and should be somewhat 
+economical to run in perpetuity. It's not unreasonable to imagine that the Tari network will grow to an order of 
+magnitude larger than any Layer One in terms of number of nodes.
```
Comment (SWvheerden) [inline _updates/2023-09-11-update-117.md]:
```suggestion
magnitude larger than any Layer One in terms of number of the nodes.
```

**Review by SWvheerden — COMMENTED** [review]
Looks good

File: _updates/2023-09-11-update-117.md (hunk 3 above)
Comment (CjS77) [reply _updates/2023-09-11-update-117.md]: Almost. It should be in terms of _the_ number of nodes.; but is also ok as written.

Comment (CjS77) [inline _updates/2023-09-11-update-117.md]:
```suggestion
magnitude larger than any Layer One in terms of the number of nodes.
```

File: _updates/2023-09-11-update-117.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+overall capacity is now 40 transactions per second.
+
+In a scenario of "full adoption", there might be 10,000 nodes on the network. This would result in 400 shards, each
+with 25 nodes. This is equivalent to 200 "cores", and the overall capacity is now 4,000 transactions per second, all the 
+while maintaining sub-second finalisation.
+
+## Closing thoughts
+
+Can Tari get to 10,000 or even more nodes? There are currently [5500 Ethereum nodes](https://www.ethernodes.org/) 
+running. Each of these must validate _every transaction_ and maintain a copy of the _entire state_ of the network.
```
Comment (CjS77) [inline _updates/2023-09-11-update-117.md]:
```suggestion
Can Tari get to 10,000 or even more nodes? Can Tari get to 10,000 or even more nodes? There are currently between 5500 and 12000 Ethereum nodes running, [depending on](https://etherscan.io/nodetracker#) [who you](https://www.ethernodes.org/) [ask](https://nodewatch.io/). Each of these must validate _every transaction_ and maintain a copy of the _entire state_ of the network. 
```

**Comment by CjS77** [comment]
ACK

**Comment by CjS77** [comment]
> Happy, just think the 5500 eth nodes figure is not from an up to date source

It depends on which tracker you use. I link to all three working ones referenced on the eth website now.

**Review by stringhandler — APPROVED** [review]

**Pull request merged**